<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrushFirm</class>
 <widget class="QWidget" name="BrushFirm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BrushFirm</string>
  </property>
  <widget class="QLabel" name="lab_productVersion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>产品型号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_num">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>串号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_nowVersion">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>现有版本号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_hardwareVersion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>硬件版本号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_softwareVersion">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>软件版本号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_newVersion">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>新版本号选择：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_other">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>其他：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_versoin">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>版本号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_commu">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>通信方式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_introduce">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>版本描述：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_port">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>串口号：</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>500</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_downlode">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>550</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_num">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>123456</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_softwareVersion">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>V1.0.0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_introduce">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_hardwareVersion">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>V1.0.0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_port">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>串口(默认115200)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_productVersion">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>AWA....</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_version">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>V1.0.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>V1.0.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>V1.1.0</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_port">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>360</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_chooseFile">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>440</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> 选择文件</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_filePath">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
